typings/react-dropzone.d.ts

```diff
@@ -8,9 +8,16 @@ export interface DropzoneProps extends DropzoneOptions {
   children?(state: DropzoneState): JSX.Element;
 }
 
+export enum ErrorCodes {
+ FILE_INVALID_TYPE = 'file-invalid-type',
+ FILE_TOO_LARGE = 'file-too-large',
+ FILE_TOO_SMALL = 'file-too-small',
+ TOO_MANY_FILES = 'too-many-files',
+}
+
```

Comment:
```suggestion
export enum ErrorCode {
 FileInvalidType = 'file-invalid-type',
 FileTooLarge = 'file-too-large',
 FileTooSmall = 'file-too-small',
 TooManyFiles = 'too-many-files',
}

```

Reply:
Constants are generally capitalized.

Why should we use pascalcase to set the `ErrorCode` options?

Reply:
TypeScript documents it as such, see [enums](https://www.typescriptlang.org/docs/handbook/enums.html).

Reply:
Good point, I did not have it in mind, thank you very much.